src/main.cpp

```diff
@@ -328,6 +333,7 @@ void getCommandSchemaJson(JsonVariant json)
   
   JsonObject channels = properties.createNestedObject("channels");
   channels["type"] = "array";
+  channels["description"] = "Used to control a specified strip";
```

Comment:
Set the levels for each channel on a specified strip. Strip is identified by the index defined when configuring the controller. Levels are a value between 0-255. Optionally set a fade interval for this command, otherwise uses the controller fade interval (defaults to 500us).